pkg/options.go

```diff
@@ -73,4 +73,7 @@ type imageCMDOptions struct {
 	WatchTimeout    time.Duration
 	//sync options
 	SpecYAML string
+	//info options
+	Filename string
+	TemDir   string
```

Comment:
I really don't see lot of benefits adding these variables part of this struct, lets create them local to that command itself.

Reply:
Removed these variables and used local variables.